lib/views/Login.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:retro_shopping/helpers/constants.dart';
+import 'package:retro_shopping/widgets/retro_button.dart';
+import 'package:retro_shopping/dashboard.dart';
+
+class LoginScreen extends StatefulWidget{
+  @override
+  State<StatefulWidget> createState() {
+    return LoginScreenState();
+  }
+}
+
+TextEditingController _emailController = TextEditingController();
+TextEditingController _passwordController = TextEditingController();
+
+class LoginScreenState extends State<LoginScreen>{
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Instead of using MediaQuery again and again, add 2 fields above this Scaffold named width and height. Like this:
```
final double height = MediaQuery.of(context).size.height;
final double width = MediaQuery.of(context).size.width;
```

Check search.dart if still didn't get it.